TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Program.cs

```diff
@@ -80,6 +81,15 @@
     app.MapHangfireDashboardWithAuthorizationPolicy(AuthorizationPolicies.AdminOnly, "/_hangfire");
 }
 
+// This endpoint is used to proxy files from external URLs (e.g. Azure Blob Storage SAS URLs) to allow us to use the filename with extension in the URL
+// which helps the browser choose an appropriate app to open the file with (in particular for Outlook .msg files).
+app.MapGet("/files/{fileName}", async (HttpClient httpClient, [FromRoute] string fileName, [FromQuery] string fileUrl) =>
```

Comment:
Can we move this to an extension method in the `Endpoints` namespace (similar to what we have for the JWKs endpoint in the API)?